### ISSUE TITLE
Support BungeeCord for Minecraft ?

### ISSUE THREAD

**Issue by Erdack54** [GameServerManagers/LinuxGSM#1282]
Hello, could create a script to manage the BungeeCord server ?

**Comment by cedarlug** [comment]
That would be hard to carry out as things are currently, because the default server is the vanilla Minecraft and not Spigot.

**Comment by Erdack54** [comment]
A good? I thought that you would have to take over the script for Minecraft servers and adapt it to BungeeCord. For example replace the command "stop" that is send to the tmux instance and send "end", his heart not? I ask her now I used LGSM to manage several servers Minecraft and I'd like to manage BungeeCord with LGSM.

Sorry for my English class zero, I'm french and I used Bing Translator

**Comment by ChaosMTA** [comment]
Spigot is planned to be supported via mods with LGSM however is BungeeCord a spigot plugin or is it a completely different Jar independent of spigot?

If it's independent it's easier to implement, if not then you will probably just have an automated spigot install and the rest is up to you.

**Comment by Erdack54** [comment]
Yes BungeeCord is a .jar and works completely differently from Spigot because it is a proxy, but it is there in no way meet LGSM BungeeCord?

**Comment by ChaosMTA** [comment]
I cannot understand your English regarding your question.

**Comment by dgibbs64** [comment]
If its just a jar you can just replace the vanilla one with the BungeeCord one. That will probably work. BUt there is not going to be an official version for LGSM as there would be countless different minecraft versions. Mod support is being investigated for Minecraft though

**Comment by Erdack54** [comment]
Yes his works simply wait 30 seconds to stop him or his works, thanks anyway

**Comment by dgibbs64** [comment]
Minecraft is being reviewed with plans to add waterfall support. BungeeCord will also be considred

**Comment by h3o66** [comment]
Added Waterfall as a PR with working updates
